src/css/showHide.js

```diff
@@ -4,6 +4,23 @@ define( [
 	"../css/var/isHidden"
 ], function( jQuery, dataPriv, isHidden ) {
 
+var defaultDisplayMap = {};
+
+function setDefaultDisplay( elem ) {
```

Comment:
If the "force visibility" behavior is here to stay, we should consider a public `jQuery.defaultDisplay( element )` interface so this function doesn't go the way of its predecessor.